base/precompile.jl

```diff
@@ -1115,28 +1115,28 @@ precompile(Tuple{typeof(Base._jl_spawn), String, Array{String, 1}, Ptr{Void}, Ba
 precompile(Tuple{Type{Base.Timer}, Int64, Float64})
 precompile(Tuple{typeof(Base.sleep), Int64})
 precompile(Tuple{typeof(Base._uv_hook_close), Base.PipeEndpoint})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{IO}}})
-precompile(Tuple{Type{Base.Nullable{IO}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{AbstractString}}})
-precompile(Tuple{Type{Base.Nullable{AbstractString}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Integer}}})
-precompile(Tuple{Type{Base.Nullable{Integer}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Union{Int64, Symbol}}}})
-precompile(Tuple{Type{Base.Nullable{Union{Int64, Symbol}}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Base.Cmd}}})
-precompile(Tuple{Type{Base.Nullable{Base.Cmd}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Any}}})
-precompile(Tuple{Type{Base.Nullable{Any}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Bool}}})
-precompile(Tuple{Type{Base.Nullable{Bool}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Base.Process}}})
-precompile(Tuple{Type{Base.Nullable{Base.Process}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Base.Dict{K, V} where V where K}}})
-precompile(Tuple{Type{Base.Nullable{Base.Dict{K, V} where V where K}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Array{T, N} where N where T}}})
-precompile(Tuple{Type{Base.Nullable{Array{T, N} where N where T}}})
-precompile(Tuple{typeof(Base.convert), Type{Base.Nullable{Base.Process}}, Base.Process})
-precompile(Tuple{typeof(Base.convert), Type{Base.Nullable{Bool}}, Bool})
+precompile(Tuple{typeof(Base.eltype), Type{Base.Union{Some{IO}, Null}}})
```

Comment:
Does it make any sense to call `precompile` with `Union` types? Or should each line be duplicated, once for `Null` and once for `Some{T}`?